crates/uv-distribution/src/error.rs

```diff
@@ -52,7 +53,7 @@ pub enum Error {
 
     // Build error
     #[error(transparent)]
-    Build(anyhow::Error),
+    Build(AnyErrorBuild),
```

Comment:
Hmmm. I believe previously, with `anyhow::Error`, this would result in printing its entire chain. But I'm not sure this new definition does? It might only emit the top of the chain when printing.